### PR TITLE
Stop calling create_queue on every instatiation of SQSinsertIntersection and SQSRamp

### PR DESCRIPTION
**Pull request by SuneKjaergaard**
This PR changes the approach of getting the SQS Queue in SQSinsertIntersection and SQSRamp.

With the current approach create_queue is called on every instantiation of these objects. This works, because it will just return the existing queue if it already exists.
The problem with this approach is that since create_queue is management event in AWS (as you're adding a new queue), it will be logged by CloudTrail, if that is enabled. And so you will bloat your cloudtrail logs.

If you also have enabled Guardduty, it will inspect all the events in cloudtrail, leading to increased cost.

In the new approach, we use the more pythonic strategy of asking for forgiveness rather than permission, and catch the error that happens, when you try to get a non-existent queue, and then create it.

### CODE REVIEW

File: motorway/contrib/amazon_sqs/intersections.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from motorway.intersection import Intersection
 import boto3
+from botocore.exceptions import Que, ClientError
```
Comment (guy881) [inline motorway/contrib/amazon_sqs/intersections.py:3]: `Que` imported but not used, is there really a class Que? :D

Comment (SuneKjaergaard) [reply motorway/contrib/amazon_sqs/intersections.py:3]: Hmm, this looks like a leftover, good catch

Comment (SuneKjaergaard) [reply motorway/contrib/amazon_sqs/intersections.py]: Removed

**Review by alesdotio — APPROVED** [review]

**Comment by SuneKjaergaard** [comment]
Want to take a look @MortenKaae ?

**Review by MortenKaae — APPROVED** [review]
Tested locally and seems to work as intended 👍🏻

**Pull request merged**